Source/Core/Core/HW/WiimoteReal/IOdarwin.mm

```diff
@@ -85,16 +98,14 @@ @interface ConnectBT : NSObject
       found_wiimotes.push_back(wm);
     }
   }
-
-  [bth release];
+  
   [bti release];
   [sbt release];
 }
 
 bool WiimoteScannerDarwin::IsReady() const
 {
-  // TODO: only return true when a BT device is present
-  return true;
+  return m_host_controller != nil;
```

Comment:
Is this even used? As far as I can tell, ``WiimoteScannerBackend::IsReady`` is called from ``WiimoteScanner::IsReady()`` but nothing ever calls that.